nvflare/app_opt/pt/pt_decomposers.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Remove this file and use pt/decomposers.py

Reply:
Done, file changes are within research/auto-fed-rl.